src/applications/pokemon_summary_screen/sprite.c

```diff
@@ -7,8 +7,7 @@
 
 #include "applications/pokemon_summary_screen/main.h"
 #include "applications/pokemon_summary_screen/subscreen.h"
-#include "overlay007/struct_ov7_0224F2EC.h"
-#include "overlay007/struct_ov7_0224F358.h"
+#include "data/pst_pal.naix"
```

Comment:
**polish**: `#include "res/data/pst_pal.naix"` will sort this to the bottom of the includes, which I think is preferable.